### PR TITLE
[ci] Add rspec test for branching packages

### PR DESCRIPTION
**Pull request by bgeuken**
Since this heavily relies on the backend, we need to create backend data
here. And clean them up afterwards.

### CODE REVIEW

File: src/api/spec/factories/packages.rb [hunk 1]
```diff
@@ -3,7 +3,12 @@
     sequence(:name) { |n| "#{Faker::Internet.domain_word}#{n}" }
     factory :package_with_file do
       after(:create) do |package|
-        Suse::Backend.put("/source/#{package.project.name}/#{package.name}/somefile.txt", Faker::Lorem.paragraph)
```
Comment (hennevogel) [inline src/api/spec/factories/packages.rb]: Hm, you're only doing this to get the right response in the cassette right?

Comment (bgeuken) [inline src/api/spec/factories/packages.rb]: Right.

Comment (bgeuken) [inline src/api/spec/factories/packages.rb]: I removed the global write through enablement and added a comment that this has to be turned on when creating new factories, or updating existing

**Comment by bgeuken** [comment]
#1835 includes this commit.

=> closing

**Pull request closed without merging**